docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -742,6 +789,20 @@ quill send message.json
 ```
 
 
+## Critical proposals
```

Comment:
I would move this on top of "generic proposals" and make it a ### title. I think it can be a sub-section of native proposals (I made another comment close to where I mean so it is hopefully more clear)